app/assets/config/thredded_manifest.js

```diff
@@ -0,0 +1,4 @@
+//= link thredded.js
+//= link thredded.css
+//= link_directory ../images/thredded .svg
```

Comment:
There are some png images as well for the onebox:
https://github.com/thredded/thredded/tree/master/app/assets/images/favicons

Reply:
thanks, done